src/transformers/models/atlas/tokenization_atlas.py

```diff
@@ -0,0 +1,120 @@
+# coding=utf-8
+# Copyright 2022, The ATLAS Authors and The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tokenization classes for ATLAS."""
+import os
+import warnings
+from typing import List, Optional
+
+from ...tokenization_utils_base import BatchEncoding
+from ...utils import logging
+from .configuration_atlas import AtlasConfig
+
+
+logger = logging.get_logger(__name__)
+
+
+class AtlasTokenizer:
```

Comment:
We have to use the super class `PreTrainedTokenizer` to fit in the library. The function that you have to re-define are included.